### PR TITLE
Add basis for cancel donation confirmation email

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Can be merged as-is as far as I'm concerned, though of course will need follow up with actual template content to complete the UC task. I'm pulling the "I don't know this German stuff" card there :)

File: src/Factories/FunFunFactory.php [hunk 1]
```diff
@@ -601,9 +601,9 @@ private function newCancelDonationMailer(): TemplateBasedMailer {
 			$this->getMessenger(),
 			new TwigTemplate(
 				$this->getTwig(),
-				 'TODO.twig' // 'Donation_Cancellation_Confirmation.twig'
+				 'Donation_Cancellation_Confirmation.twig'
```
Comment (KaiNissen) [inline src/Factories/FunFunFactory.php]: are you trying to sneak a whitespace in here?

Comment (JeroenDeDauw) [inline src/Factories/FunFunFactory.php]: confoose

**Pull request merged**